x11-libs/wxGTK/wxGTK-3.0.4-r5.ebuild

```diff
@@ -53,6 +53,13 @@ PATCHES=(
 )
 
 multilib_src_configure() {
+	# ideally we want !tc-ld-is-bfd for best future-proofing, but it needs
+	# https://github.com/gentoo/gentoo/pull/28355
+	# mold needs this too but right now tc-ld-is-mold is also not available
+	if tc-ld-is-lld; then
```

Comment:
Please use the new way of writing this: `append-ldflags $(test-flags-CCLD -Wl,--undefined-version)`.

Existing packages have already been updated.

Reply:
Also, add a "bug #123456" ref in the comment.

Reply:
Thank you for review! Please note that there are a bunch of packages with old block: https://github.com/search?q=repo%3Agentoo%2Fgentoo%20%22ideally%20we%20want%22&type=code

Reply:
I fixed most of those in ~arch but not in stable fwiw